include/lsst/afw/table/Schema.h

```diff
@@ -272,6 +272,17 @@ class Schema {
     /// @brief Copy constructor.
     Schema(Schema const & other);
 
+    /// @brief Construct from reading a FITS file.
+    ///
+    /// Reads from the nominated 'hdu' (1=PHU which cannot be a catalog).
+    static Schema readFits(std::string const& filename, int hdu=2);
```

Comment:
`Catalog::readFits` uses a default of `hdu=0`, where 0 is interpreted as "the first HDU with `NAXIS != 0`.